scripts/updateLinksCatalog.py

```diff
@@ -0,0 +1,40 @@
+import os
+import re
+
+from model import Link
+from main import loadEnvFile
+from managers import DBManager
+
+
+def main():
+    '''
+    Updating NYPL Link media type with new link to replace text/html link
+    '''
+
+    loadEnvFile('local-qa', fileString='config/{}.yaml')
+
+    dbManager = DBManager(
+        user= os.environ.get('POSTGRES_USER', None),
+        pswd= os.environ.get('POSTGRES_PSWD', None),
+        host= os.environ.get('POSTGRES_HOST', None),
+        port= os.environ.get('POSTGRES_PORT', None),
+        db= os.environ.get('POSTGRES_NAME', None)
+    )
+
+    dbManager.generateEngine()
+
+    dbManager.createSession()
+
+    for link in dbManager.session.query(Link) \
+        .filter(Link.media_type == 'text/html').all():
+            
+            if link.flags != None and len(link.flags) > 0:
+                for i in link.flags:
+                    if i['catalog'] == True:
+                        link.media_type = 'application/html/catalog'
```

Comment:
This totally works, but it might be more efficient to do an `update` query as described here: https://docs.sqlalchemy.org/en/14/core/dml.html#sqlalchemy.sql.expression.update

Alternatively, should be able to add the catalog `True` check as a filter, which will simplify this a bit.